assets/app-test-zendesk.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
not fully functional, an illustration of a manifest bundled in a local file, should probably be updated to something that works